src/workloads/docs/Generators.yml

```diff
@@ -16,6 +16,7 @@ Owner: "@mongodb/stm"
 # 	"string1" : "T336d",
 # 	"string2" : "B2A54",
 # 	"string3" : "LChkGtHKBI",
+#   "counter" : 1008,
```

Comment:
[nit] indentation.

Reply:
Fixed. It was a combination of spaces and tabs that showed right in VS Code - but not in github... Now made it same way as other lines.